app/models/course.rb

```diff
@@ -282,6 +282,10 @@ def tracked_revisions
              .where(wiki_id: wiki_ids)
   end
 
+  def tracked_namespaces
+    return ['0']
```

Comment:
Should this be an integer instead of a string?

Reply:
https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/dbf2c87e75c2cbfeafb552986eff16ea563f62a3/app/models/course_data/articles_courses.rb#L131-L133
Yeah, but in articles_courses it was a string, so I kept it a string.

Reply:
Ah. I don't think there's any reason for it to be a string, as namespace is an integer in the database.

Reply:
okay.